.travis.yml

```diff
@@ -19,17 +19,28 @@ matrix:
       script:
         - sudo ./travis-scripts/valgrind.sh
 
+    # JOB
```

Comment:
just delete it? maybe leave a comment about why NOT to test against master binaries?

Reply:
k